doc-site/docs/tutorials/private-storage.md

```diff
@@ -0,0 +1,147 @@
+
+---
+
+## Prerequisites
+
+Before starting, make sure you have:
+
+1. Completed the [Public Storage Tutorial](./public-storage.md) and are familiar with:
+   - Deploying and interacting with contracts.
+   - Using Paladin SDK for blockchain transactions.
+2. A running Paladin network with multiple nodes (at least 3 for this tutorial).
+
+---
+
+## Overview
+
+The `PrivateStorage` tutorial demonstrates how to:
+
+1. Create a **privacy group** with selected members.
+2. Deploy a **private Storage contract** within the group.
+3. Interact with the contract securely within the group.
+4. Test privacy by attempting access from a non-member node.
+
+The example code can be found in the [Paladin example repository](https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/example/private-storage).
+
+The Solidity contract remains the same as in the [Public Storage Tutorial](https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/solidity/contracts/tutorial/Storage.sol). However, the interaction is scoped to the privacy group.
+
+---
+
+## Step 1: Create a Privacy Group
+
+To enable private interactions, start by creating a privacy group with selected members.
+
+```typescript
+// Create a privacy group with Node1 and Node2
+logger.log("Creating a privacy group for Node1 and Node2...");
+const penteFactory = new PenteFactory(paladinNode1, "pente");
+const memberPrivacyGroup = await penteFactory.newPrivacyGroup(verifierNode1, {
+  group: {
+    salt: newGroupSalt(), // Generate a unique salt for the group
+    members: [verifierNode1, verifierNode2], // Include Node1 and Node2 as members
+  },
+  evmVersion: "shanghai",
+  endorsementType: "group_scoped_identities",
+  externalCallsEnabled: true,
+});
+
+if (!checkDeploy(memberPrivacyGroup)) {
+  logger.error("Failed to create the privacy group.");
+  return false;
+}
+logger.log("Privacy group created successfully!");
+```
+
+---
+
+## Step 2: Deploy the Contract in the Privacy Group
+
+Deploy the `Storage` contract within the created privacy group.
+
+```typescript
+logger.log("Deploying a private Storage contract...");
+const contractAddress = await memberPrivacyGroup.deploy(
+  storageJson.abi, // ABI of the Storage contract
+  storageJson.bytecode, // Bytecode of the Storage contract
+  verifierNode1 // Deploying as Node1
+);
+
+if (!contractAddress) {
+  logger.error("Failed to deploy the private Storage contract.");
+  return false;
+}
+logger.log(`Private Storage contract deployed! Address: ${contractAddress}`);
+```
+
+---
+
+## Step 3: Store and Retrieve Values as Group Members
+
+### Store a Value
+
+Group members can store values securely in the private contract.
+
+```typescript
+const privateStorage = new PrivateStorage(memberPrivacyGroup, contractAddress);
+
+logger.log("Storing value (125) in the private Storage contract...");
+const storeTx = await privateStorage.invoke(verifierNode1, "store", { num: 125 });
```

Comment:
We had talked offline about adjusting the signature of the Pente helpers, and replacing `invoke` here with a `sendTransaction` that feels a bit more similar to the public version. Did you want to do that, or want some help with it?